### PR TITLE
DOCSP-26370: FAQ entry for MongoSecurityException

### PR DESCRIPTION
**Pull request by ccho-mongodb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-26370
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-26370-faq-mongosecurityexception/faq/#how-do-i-prevent-the--com.mongodb.mongosecurityexception--error-

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/faq.txt [hunk 1]
```diff
@@ -171,6 +171,28 @@ your classpath, check the following items in your environment:
    :ref:`API documentation <java-api-landing>`.
 
 
+How do I prevent the "com.mongodb.MongoSecurityException" error?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Your application may throw this exception if you attempt to connect to
```
Comment (mongoKart) [inline source/faq.txt:177]: I: [may/might](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#m:~:text=than%20matrices.-,may,-Use%20may%20to)

```suggestion
Your application might throw this exception if you attempt to connect to
```

File: source/faq.txt [hunk 2]
```diff
@@ -171,6 +171,28 @@ your classpath, check the following items in your environment:
...
+a MongoDB deployment by specifying invalid or incorrectly formatted credentials.
```
Comment (mongoKart) [inline source/faq.txt:178]: S: simplification

```suggestion
Your application may throw this exception if you specify invalid or incorrectly formatted credentials
when connecting to a MongoDB deployment.
```

File: source/faq.txt [hunk 3]
```diff
@@ -171,6 +171,28 @@ your classpath, check the following items in your environment:
...
+
+If you receive this error when you attempt to connect to a MongoDB deployment,
+check the following items in your code:
+
+- The connection URI corresponds to the correct MongoDB deployment.
+  To learn more about setting your connection URI, see :ref:`connection-uri`.
+
+- The credentials for the authentication mechanism that you specified are
```
Comment (mongoKart) [inline source/faq.txt:186]: S: On line 191, you leave out "that" in the first sentence. Suggest being consistent across list items.

**Review by mongoKart — APPROVED** [review]
One small change and a couple suggestions, but otherwise LGTM

**Review by stIncMale — APPROVED** [review]

**Pull request merged**